components/package.json

```diff
@@ -30,26 +30,29 @@
         "@types/color": "^3.0.1",
         "@types/enzyme": "^3.10.4",
         "@types/enzyme-adapter-react-16": "^1.0.5",
-        "@types/jest": "^27.4.1",
-        "@types/react-dom": "^17.0.11",
+        "@types/jest": "^27.0.0",
+        "@types/react-dom": "^18.0.5",
         "@typescript-eslint/eslint-plugin": "^5.14.0",
         "@typescript-eslint/parser": "^5.14.0",
         "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
+        "cheerio": "1.0.0-rc.10",
         "clsx": "^1.1.0",
         "enzyme": "^3.5.0",
         "enzyme-adapter-react-16": "^1.8.0",
         "eslint": "^8.15.0",
         "eslint-config-prettier": "^8.5.0",
         "eslint-plugin-react": "^7.29.4",
-        "jest": "^27.5.1",
+        "i": "^0.3.7",
+        "jest": "^27.0.0",
+        "npm": "^8.12.1",
         "prettier": "^2.2.0",
         "react": "^18.0.0",
         "react-dom": "^18.0.0",
-        "ts-jest": "^27.1.3",
+        "ts-jest": "^28.0.4",
         "typescript": "^4.2.3"
     },
     "peerDependencies": {
-        "@brightlayer-ui/colors": "^3.0.0",
+        "@brightlayer-ui/colors": "3.0.1",
```

Comment:
I'm pretty sure you can keep the `peerDependencies` colors@^3.0.0 since we don't need to lock down any adopters version.